### PR TITLE
Update docs for the tokenUrl config and add a manual test

### LINKED ISSUE #16: Update docs for the tokenUrl config and add a manual test
The `tokenUrl` config can be now a callback to the token value.

### LINKED ISSUE #20: CloudServices plugin misses the `pluginName`.
...So it can't be required without an import.

I'll make it in the https://github.com/ckeditor/ckeditor5-cloud-services/pull/17.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Enhanced docs for the tokenUrl config. Closes #16. Closes #20.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

**Comment by pjasiun** [comment]
We need a manual test for it.

Also, we should mention it in docs.

**Comment by pjasiun** [comment]
I think docs should be somewhere here: https://ckeditor.com/docs/cs/latest/guides/token-endpoints/tokenendpoint.html.

**Comment by ma2ciek** [comment]
Eventually, we could have something in the `ckeditor5-cloud-services/docs` too as it's empty right now

**Comment by ma2ciek** [comment]
Unfortunately, I don't remember how we agreed on the manual test. I create a pretty basic one, but I can add some inputs to e.g enable setting some custom headers.

**Comment by pjasiun** [comment]
> Actually, I'm not sure where to put these docs exactly. After the `Example token payload` section?

Yep. I think it will be a good place. You can mention how the default requests to the server look like and how you can configure it.

**Comment by ma2ciek** [comment]
☝️ This would be better IMHO.

**Comment by pjasiun** [comment]
> Unfortunately, I don't remember how we agreed on the manual test.

We talked that this manual test could use some not-blocking dialog in the token handler which will let you accept the the request or reject it. This way we will be able to check if the editor is waiting for the promice to be resolved in the proper way. It will also test error handling (the situation when the request is rejected).

File: tests/manual/token.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		plugins: [ ArticlePluginSet, CloudServices ],
+		toolbar: [ 'heading', '|', 'undo', 'redo' ]
+	} )
+	.then( editor => {
+		window.editor = editor;
+
+		const cloudServices = editor.plugins.get( CloudServices );
+
+		refreshTokenButton.addEventListener( 'click', () => {
+			cloudServices.token._refreshToken();
```
Comment (pjasiun) [inline tests/manual/token.js]: I am against using protected API in the manual test. One can refresh the page to test it again.

**Comment by ma2ciek** [comment]
Note that we should release https://github.com/ckeditor/ckeditor-cloud-services-core with https://github.com/ckeditor/ckeditor-cloud-services-core/pull/17 and update the `package.json` to make the manual test working.

**Comment by ma2ciek** [comment]
Docs are on the https://github.com/cksource/bigbang-docs.ckeditor.com/tree/cf-it-16

**Pull request merged**